test/unit/lib/mapResults.js

```diff
@@ -2,8 +2,9 @@ const chai = require('chai');
 const VError = require('verror');
 
 const mapResults = require('../../../app/lib/mapResults');
-const types = require('../../../app/lib/constants').types;
+const constants = require('../../../app/lib/constants');
```

Comment:
This could be changed to used object destructuring e.g. `const { metrics, types } = require('../../../app/lib/constants');` which would mean line 7 could be removed and lines 90 & 91 become shorter although line 89 would need the variable `metrics` to be renamed.